content/docs/components/misc/metadata.md

```diff
@@ -4,43 +4,172 @@ description = "Tracking and managing metadata of machine learning workflows in K
+  to capture metadata for the metrics used to evaluate an ML model.
+* [`model.json`](https://github.com/kubeflow/metadata/blob/master/schema/alpha/artifacts/model.json)
+  to capture metadata for an ML model that your workflow produces.
+
+<a id="metadata-ui"></a>
+## Tracking artifacts on the Metadata UI
+
+You can view a list of logged artifacts and the details of each individual 
+artifact in the **Artifact Store** on the Kubeflow UI.
+
+1. Go to Kubeflow in your browser. (If you haven't yet opened the 
+  Kubeflow UI, find out how to [access the
+  Kubeflow UIs](https://www.kubeflow.org/docs/other-guides/accessing-uis/).)
+1. Click **Artifact Store** in the left-hand navigation panel:
+  <img src="/docs/images/metadata-ui-option.png" 
+    alt="Metadata UI"
+    class="mt-3 mb-3 border border-info rounded">
+
+1. The **Artifacts** screen opens and displays a list of items for all the
+  metadata events that your workflows have logged. You can click the name of 
+  each item to view the details. 
+  
+    The following examples show the items that appear when you run the 
+    `demo.ipynb` notebook described [above](#demo-notebook):
+
+    <img src="/docs/images/metadata-artifacts-list.png" 
+    alt="A list of metadata items"
+    class="mt-3 mb-3 border border-info rounded">
+
+  * Example of **model** metadata with the name "MNIST":
+
+        <img src="/docs/images/metadata-model.png" 
+          alt="Model metadata for an example MNIST model"
+          class="mt-3 mb-3 border border-info rounded">
+
+  * Example of **metrics** metadata with the name "MNIST-evaluation":
+
+        <img src="/docs/images/metadata-metrics.png" 
+          alt="Metrics metadata for an evaluation of an MNIST model"
+          class="mt-3 mb-3 border border-info rounded">
+
+  * Example of **dataset** metadata with the name "mytable-dump":
+
+        <img src="/docs/images/metadata-dataset.png" 
+          alt="Dataset metadata"
+          class="mt-3 mb-3 border border-info rounded">
+
 
-You can find an example of how to use the logging API in this [notebook](https://github.com/kubeflow/metadata/blob/master/sdk/python/demo.ipynb).
 
-## Backend
+## Backend and REST API
 
-The backend uses [ML-Metadata](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) to manage all the metadata and relations. It exposes a [REST API](https://github.com/kubeflow/metadata/blob/master/api/service.swagger.json).
+The Kubeflow metadata backend uses [ML Metadata
+(MLMD)](https://github.com/google/ml-metadata/blob/master/g3doc/get_started.md) 
+to manage the metadata and relationships. 
+
+The backend exposes a 
+[REST API](https://github.com/kubeflow/metadata/blob/master/api/service.swagger.json).
+
+You can add your own metadata types so that you can log metadata for custom
+artifacts. To add a custom type:
+
+* Define your custom metadata type using the same 
+  [schema format](https://github.com/kubeflow/metadata/tree/master/schema) as 
```

Comment:
This is not longer true: the customized schema is different from the predefined one.

We should encourage the reader to follow the API reference to construct the REST queries.

Maybe after https://github.com/kubeflow/website/pull/1058 is merged, we can add a link to the corresponding query.

Reply:
Thanks! Done.